src/incflo_advance.cpp

```diff
@@ -82,5 +82,9 @@ void incflo::Advance()
     {
         amrex::Print() << "Time per step " << end_step << std::endl;
     }
+
+#ifdef INCFLO_USE_PARTICLES
```

Comment:
Wouldn't we want to include this redistribution in the calculation of the time per step?

Reply:
Done